docs/documentation.md

```diff
@@ -252,11 +248,11 @@ This example shows you how to create a very simple search results page with a se
 <div class="codebox-combo">
 
 <img class="widget-icon pull-left" src="../img/icon-widget-searchbox.svg">
-The search box is the widget to use for text input. Underneath, it will call the [Algolia API](https://www.algolia.com/doc/rest) and return results based on your [configured relevance options](https://www.algolia.com/doc/tutorials/ranking-formula).

```

Comment:
I rewrote this part to make it clearer. I felt like talking about the query parameter and attributeToIndex was too broad.